src/caret_analyze/plot/visualize_lib/bokeh/util/util.py

```diff
@@ -77,7 +78,7 @@ def init_figure(
 
     return figure(
         frame_height=270, frame_width=800, title=title, y_axis_label=y_axis_label or '',
-        x_axis_label=x_axis_label, tools=tools, active_scroll=active_scroll
+        x_axis_label=x_axis_label or x_axis_label_, tools=tools, active_scroll=active_scroll
```

Comment:
This description is not intuitive.
Is it possible to implement the following? (in Line65)

```python
if x_axis_label is None:
  if x_axis_type == 'system_time':
    x_axis_label = 'system time [s]'
  elif xaxis_type == 'sim_time':
    x_axis_label = 'simulation time [s]'
  else:
    x_axis_label = xaxis_type

...

x_axis_label=x_axis_label, tools=tools, active_scroll=active_scroll

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/289/commits/478cc692684d3e55acdb70a5b6626116c9f72d66